oracle-scripts/src/script.rs

```diff
@@ -6,6 +6,9 @@ const DATA_SOURCE_GITHUB: i64 = 63;
 const DATA_SOURCE_DISCORD: i64 = 62;
 const DATA_SOURCE_TWITCH: i64 = 60;
 const DATA_SOURCE_DOMAIN: i64 = 67;
+const DATA_SOURCE_YOUTUBE: i64 = 101;
+const DATA_SOURCE_TELEGRAM: i64 = 102;
+const DATA_SOURCE_INSTAGRAM: i64 = 0;
```

Comment:
We didn't upload the Instagram data source to main net since I don't have mainnet deployer key.

Reply:
I think we can wait until the data source has been deployed to merge this.

Reply:
Updated